main.js

```diff
@@ -8,21 +8,40 @@ import Session from './Session.js';
 
 export {default as SessionService} from './SessionService.js';
 
+/**
+ * Creates a Session with an optional id or store.
+ *
+ * @param {object} [options={}] - Options to use.
+ * @param {string} [options.id='session.default'] - An id for the session.
+ * @param {object} [options.store=defaultStore] - A store for the session.
+ *
+ * @returns {object} The created session.
```

Comment:
please annotate `@returns {Promise<object>}` here and throughout as appropriate.  It appears that the automatic doc generation thing does not properly detect async functions.  Do you know different?

Reply:
https://github.com/digitalbazaar/bedrock-web-session/pull/8/commits/27e231909b49977911f48b5b54a854fa88033063

The above addresses it.